docs/data/tree-view/rich-tree-view/editing/editing.md

```diff
@@ -0,0 +1,86 @@
+---
+productId: x-tree-view
+title: Rich Tree View - Editing
+githubLabel: 'component: tree view'
+packageName: '@mui/x-tree-view'
+waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/treeview/
+packageName: '@mui/x-tree-view'
+---
+
+# Rich Tree View - label editing
+
+<p class="description">Learn how to edit the label of Tree View items.</p>
+
+:::warning
+this doc page is stip WIP
+:::
+
+## Enable editing on certain items
```

Comment:
```suggestion
## Enable editing
```

I think we should either

1. Add a prop `labelEditing?: boolean` and if `isItemEditable` is not defined then all items are editable
2. Keep the current DX, in which case we rename this doc section "Enable label editing" 

Because right now I feel like we miss the most basic use case: how to enable label editing for every item :laughing:

Reply:
I renamed this section for now, but if we feel like we want to introduce another prop, I'm fine with that too 👌